tests/get_or_create/tests.py

```diff
@@ -413,3 +417,56 @@ def test_update_callable_default(self):
             defaults={'birthday': lambda: date(1943, 2, 25)},
         )
         self.assertEqual(obj.birthday, date(1943, 2, 25))
+
+
+class UpdateOrCreateTransactionTests(TransactionTestCase):
+
+    available_apps = ['get_or_create']
+
+    @skipUnlessDBFeature('has_select_for_update')
+    @skipUnlessDBFeature('supports_transactions')
+    def test_updates_in_transaction(self):
+        """
+        Should select and update objects in a transaction to avoid race conditions. This test basically forces
```

Comment:
Please wrap docstrings at 79 characters. Ticket references are only needed for obscure issues that benefit from the additional context of the ticket. Not sure that's the case here.